packages/ui/src/stories/formkit/Slider.stories.ts

```diff
@@ -0,0 +1,28 @@
+import { StoryFn } from '@storybook/vue3'
```

Comment:
<img width="677" alt="image" src="https://github.com/user-attachments/assets/5812c8d0-5cfd-4ca0-9f10-e5ed0e201c17" />

<img width="441" alt="image" src="https://github.com/user-attachments/assets/c1bc2a3c-8190-44fc-800f-6e261b6a8aa9" />

Style is slightly different, not sure why. When you hold thumb - selection area is displaced.